src/pages/guides/javascript/how-to-transfer-spl-tokens-on-solana.md

```diff
@@ -0,0 +1,203 @@
+---
+title: How to Send and Transfer SPL Tokens on Solana
+metaTitle: How to Send and Transfer SPL Tokens on Solana
+description: Learn how to send and transfer SPL Tokens via javascript on the Solana blockchain wih Metaplex packages.
+# remember to update dates also in /components/guides/index.js
+created: '06-16-2024'
+updated: '06-24-2024'
+---
+
+This guide will show you how to build out a Javascript function to send and transfer SPL tokens on the Solana blockchain utilizing the Metaplex Umi client wrapper and MPL Toolbox packages.
+
+## Prerequisite
+
+- Code Editor of your choice (recommended Visual Studio Code)
+- Node 18.x.x or above.
+
+## Initial Setup
+
+### Initializing
+
+Start by initializing a new project (optional) with the package manager of your choice (npm, yarn, pnpm, bun) and fill in required details when prompted.
+
+```js
+npm init
+```
+
+### Required Packages
+
+Install the required pacakges for this guide.
+
+{% packagesUsed packages=["umi", "umiDefaults" ,"toolbox"] type="npm" /%}
+
+```js
+npm i @metaplex-foundation/umi
+```
+
+```js
+npm i @metaplex-foundation/umi-bundle-defaults
+```
+
+```js
+npm i @metaplex-foundation/mpl-toolbox;
+```
+
+### Imports and Wrapper Function
+
+Here we will define all needed imports for this particular guide and create a wrapper function where all our code will execute.
+
+```ts
+import { mplToolbox, transferSol } from '@metaplex-foundation/mpl-toolbox'
+import {
+  generateSigner,
+  publicKey,
+  signerIdentity,
+  sol,
+} from '@metaplex-foundation/umi'
+import { createUmi } from '@metaplex-foundation/umi-bundle-defaults'
+import { base58 } from '@metaplex-foundation/umi/serializers'
+
+// Create the wrapper function
+const transferSolana = async () => {
+  ///
+  ///
+  ///  all our code will go in here
+  ///
+  ///
+}
+
+// run the wrapper function
+transferSolana()
+```
+
+## Setting up Umi
+
+This example is going to run through setting up Umi with a `generatedSigner()`. If you wish to try this example with React you'll need to setup Umi via the `React - Umi w/ Wallet Adapter` guide. Apart from the the wallet setup this guide will for fileStorage keys and wallet adapter.
```

Comment:
Does this guide already exist? In that case we should link it. Or are you referring to https://developer-o8s1bh38b-metaplex-foundation.vercel.app/umi/connecting-to-umi#connecting-w-wallet-adapter ?

Reply:
That was one I was going to write but unless I stop writing guides this guide update is never going to go out. So for now we can either link above or we can remove this part for time being.